systemdev/testing-locally.rst

```diff
@@ -125,26 +131,77 @@ In both cases the chroot will be created by running the following command::
 
     sudo sbuild-createchroot --components=main,restricted,universe --extra-repository="deb http://archive.ubuntu.com/ubuntu/ ${chroot_distro}-updates main restricted universe" --include=ccache "${chroot_distro}" "${chroot_base}" http://archive.ubuntu.com/ubuntu/
 
-A chroot for cross-building arm64 packages on an amd64 host can e.g. be created under the directory ``/srv/chroot/ubports-${chroot_distro}-arm64`` using::
+The UBports package repository needs to be added using::
+
+    wget 'http://repo.ubports.com/keyring.gpg' -O - | sudo tee "${chroot_base}/usr/share/keyrings/ubports-keyring.gpg" >/dev/null
+    printf 'deb [signed-by=/usr/share/keyrings/ubports-keyring.gpg] %s %s main\n' "${chroot_repo}" "${chroot_distro}" | sudo tee "${chroot_base}/etc/apt/sources.list.d/ubports.list" >/dev/null
+
+Synchronizing package index files and subsequent package upgrades can be performed using::
+
+    sbuild-update -u -d ${chroot_distro}
+    
+A build can be started from inside the debianized package source directory using::
+
+    sbuild -d ${chroot_distro}
 
-    chroot_base=/srv/chroot/ubports-${chroot_distro}-arm64
-    sudo sbuild-createchroot --arch=arm64 --foreign --components=main,restricted,universe --extra-repository="deb http://ports.ubuntu.com/ubuntu-ports/ ${chroot_distro}-updates main restricted universe" --include=ccache "${chroot_distro}" "${chroot_base}" http://ports.ubuntu.com/ubuntu-ports/
+If the build was successful, the binary packages will be placed in the parent directory.  The build log will be placed inside ``~/logs``.  In case the build failed, the chroot can be inspected using::
+
+    sbuild-shell <distribution>
+    
+Cross compile build
+^^^^^^^^^^^^^^^^^^^    
+The only difference with the native build comes in the command for starting the build which is the following::
 
-For cross-building armhf packages the above command can be used with ``arm64`` changed to ``armhf``.
+    sbuild --host=arm64 --build=amd64 -d "${chroot_distro}"
+
+If the build was successful, the binary packages will be placed in the parent directory.  The build log will be placed inside ``~/logs``.  In case the build failed, the chroot can be inspected using::
+
+    sbuild-shell <distribution>
```

Comment:
I don't think we need to repeat the hint about logs and shell here again which is just a few lines above in the native build as well.

Reply:
sure, i repeated it in case someone just skips and jumps on specific part

Reply:
> I don't think we need to repeat the hint about logs and shell here again which is just a few lines above in the native build as well.

solved